sph/include/sph/hydro_ve/momentum_energy_kern.hpp

```diff
@@ -69,11 +68,9 @@ HOST_DEVICE_FUN T avRelVelCorrection(util::array<Tc, 3> R, Tc dist, Th hi, Th hj
     if (eta_ab < eta_crit) { dmy3 = std::exp(-math::pow((eta_ab - eta_crit) / T(0.2), 2)); }
     T phi_ab = T(0.5) * stl::max(T(0), stl::min(T(1), T(4) * A_ab / math::pow(T(1) + A_ab, 2))) * dmy3;
 
-    auto vAVi = Vi - phi_ab * gradV_i_dot_R;
-    auto vAVj = Vj + phi_ab * gradV_j_dot_R;
-    T    rv   = dot(R, vAVi - vAVj);
-
-    return rv;
+    // additive AV correction to rv = dot(R, Vij)
+    T rv_AV = dot(R, -phi_ab * (gradV_i_dot_R + gradV_j_dot_R));
```

Comment:
Shouldn't be a minus here?: 
gradV_i_dot_R - gradV_j_dot_R 

instead of:
gradV_i_dot_R + gradV_j_dot_R